replay_processing/sc2files.py

```diff
@@ -95,10 +94,11 @@ def extract_player_data(result, player, logger, debug=False):
     return data
 
 
-def dump(data):
+def dump(data, outfile):
```

Comment:
I only state this because our bottleneck is in reading and processing the files, I don't mind this approach. just wanted to comment.